kanban/api/serializers.py

```diff
@@ -0,0 +1,109 @@
+from kanban.models.issue import Issue
+from kanban.models.project import Project
+from rest_framework import serializers
+from rest_framework.reverse import reverse
+from user_account.api.serializers import UserAccountSerializer, AssignUserSerializer
+from user_account.models.user_account import UserAccount
+
+
+class IssueSerializer(serializers.ModelSerializer):
+    owner = serializers.HiddenField(
+        default=serializers.CreateOnlyDefault(serializers.CurrentUserDefault())
+    )
+    owner_email = serializers.CharField(source="owner.email", read_only=True)
+    project_name = serializers.CharField(source="project.name", read_only=True)
```

Comment:
I mentioned it before, usually you don't want to return strings from objects, but serialized objects themselves so it's easy to access additional properties and you have ids which you'll use for updates